lib/entities/applications/index.ts

```diff
@@ -27,4 +27,14 @@ export class Applications {
       return res;
     });
   }
+
+  public async getAll(_planIdentifier:string): Promise<IApplication[] | undefined> {
```

Comment:
Since this is publically exposed we want to add some tsdoc comments like the other method in this file. It helps users know what the method does and what it expects. Additionally, we can rename `_planIdentifier` to `planIdentifier` as it is being used by the function. In typescript when a parameter is not used, we can add a `_` to the start of it to stop typescript from checking it.

Reply:
Whoops those comments didn't get ported over, I've readded them now and have removed the underscore from the param